packages/discord.js/src/index.js

```diff
@@ -187,3 +187,4 @@ exports.WebSocket = require('./WebSocket');
 __exportStar(require('discord-api-types/v10'), exports);
 __exportStar(require('@discordjs/builders'), exports);
 __exportStar(require('@discordjs/rest'), exports);
+__exportStar(require('@discordjs/util'), exports);
```

Comment:
Is this line necessary? `@discordjs/builders` already `export *`s `@discordjs/util`